subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -143,17 +147,39 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	dp->coding_format = coding_format;
 	dp->company_id    = company_id;
 
-	/* TBD dp->sync_delay    = controller_delay; ?*/
+	/* TODO dp->sync_delay    = controller_delay; ?*/
+
+	cig = cis->group;
 
 	if (path_dir == BT_HCI_DATAPATH_DIR_HOST_TO_CTLR) {
+		burst_number = cis->lll.tx.burst_number;
+		if (cig->lll.role) {
+			/* periheral */
+			sdu_interval = cig->p_sdu_interval;
```

Comment:
Would the code overall benefit from changing `p_sdu_interval`/`c_sdu_interval` to `tx_pdu_interval`/`rx_pdu_interval`, i.e., do the if/else on the role when storing the parameters instead of when using them?

Reply:
Potentially, but not sure this needs to be done in many places, so better leave it as was thought in parameters for HCI functions

Reply:
"A framed PDU with non-zero length shall contain one or more segments. A
framed PDU with zero length can be used as a padded PDU. Padding is
required when the data does not add up to the configured number of PDUs that
are specified by the BN parameter per BIS or CIS event."

Reply:
Ack. We can always refactor if/when there is a clear benefit.